selenium/Tests/Help.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using OpenQA.Selenium;
+using System.Windows.Forms;
+using OpenQA.Selenium.Interactions;
+using System;
+using OpenQA.Selenium.Support.UI;
+
+namespace SeleniumTests.Tests
+{
+    static class Help
```

Comment:
Maybe rename this class to `WebDriverHelpers` to better express what's going on here.